torch/fx/symbolic_trace.py

```diff
@@ -209,11 +227,8 @@ def symbolic_trace(root : torch.nn.Module, delegate_class=DefaultDelegate) -> Gr
     orig_call = torch.nn.Module.__call__
 
     def module_call_wrapper(mod, *args, **kwargs):
-        if not delegate.is_leaf_module(mod):
-            return orig_call(mod, *args, **kwargs)
-        else:
-            target = _find_module(root, mod)
-            return _create_proxy(delegate, 'call_module', target, args, kwargs)
+        target = _find_module(root, mod)
```

Comment:
I guess we can void passing orig_call by unpatching `Module.__call__` for the duration of `call_module`.  Then you can must `m(*args, **kwargs)`

Reply:
Hm, I tried that but I think we actually need to make this more reentrant. When `call_module` calls `m`, inside that call the `module_call_wrapper` needs to be reinstated. Otherwise nothing beyond the first level of calls will go through `module_call_wrapper`